### PR TITLE
Python3

### PR DESCRIPTION
**Pull request by samayNathani**

### CODE REVIEW

File: detailed_search_autonomy.py [hunk 1]
```diff
@@ -20,7 +20,7 @@
 # Callback function for messages from GCS, parses JSON message and sets globals
 def xbee_callback(message, autonomyToCV):
     address = message.remote_device.get_64bit_addr()
-    msg = json.loads(message.data.decode("utf8"))
+    msg = json.loads(message.data)
```
Comment (nmcobb) [inline detailed_search_autonomy.py]: What changed here? why aren't we decoding this anymore?

File: autonomy.py [hunk 2]
```diff
@@ -311,17 +312,15 @@ def update_thread(vehicle, address, autonomyToCV):
...
-
     change_status("ready")
 
 
 # Continuously sends message to given address until acknowledgement message is recieved with the corresponding ackid.
 def send_till_ack(address, msg, msg_id):
     # Instantiate a remote XBee device object to send data.
-    send_xbee = RemoteXBeeDevice(xbee, address)
-    packed_data = packb(json.dumps(ack), use_bin_type = True)
+    send_xbee = RemoteXBeeDevice(xbee, XBee64BitAddress.from_hex_string(address))
```
Comment (nmcobb) [inline autonomy.py]: XBee64BitAddress.from_hex_string(address)) to a local variable, this is pretty redundant

File: detailed_search_autonomy.py [hunk 3]
```diff
@@ -59,11 +59,13 @@ def xbee_callback(message, autonomyToCV):
...
 def orbit_poi(vehicle, poi, configs):
     waypoints = []  # waypoints in LLA
+    altitude = configs["altitude"]
     poi_scan_altitude = configs["detailed_search_specific"]["poi_scan_altitude"]
     waypoint_tolerance = configs["waypoint_tolerance"]
     radius = configs["radius"]  # radius of circle travelled
     orbit_number = configs["orbit_number"]  # how many times we repeat orbit
-    x, y, z = conversions.geodetic2ecef(poi.lat, poi.lon, configs["altitude"])  # LLA -> ECEF
+    x, y, z = geodetic2ecef(poi.lat, poi.lon, poi_scan_altitude)  # LLA -> ECEF
+    n, e, d = ecef2ned(x, y, z, poi.lat, poi.lon, poi_scan_altitude)  # ECEF -> NED
```
Comment (nmcobb) [inline detailed_search_autonomy.py]: Should these changes be on this PR? or is this different...?

File: quick_scan.py [hunk 4]
```diff
@@ -71,10 +71,6 @@ def quick_scan(gcs_timestamp = 0, connection_timestamp = 0):
         autonomyToCV.xbeeMutex.release()
```
Comment (nmcobb) [inline quick_scan.py]: We'll look into this later

File: quick_scan_cv.py [hunk 5]
```diff
@@ -7,6 +7,7 @@
 ballTargetImgIndex = 0
 
 def quick_scan_cv(configs, autonomyToCV, gcs_timestamp, connection_timestamp):
+    return None
```
Comment (nmcobb) [inline quick_scan_cv.py]: ???

File: detailed_search_autonomy.py (hunk 1 above)
Comment (samayNathani) [reply detailed_search_autonomy.py]: Python3 does not require .decode() anymore

File: detailed_search_autonomy.py (hunk 3 above)
Comment (samayNathani) [reply detailed_search_autonomy.py]: these were from a pull

**Pull request merged**